tsconfig.json

```diff
@@ -10,9 +10,11 @@
         "moduleResolution": "node",
         "target": "es5",
         "allowJs": true,
-        "jsx": "react"
+        "jsx": "react",
+        "baseUrl": "."
     },
     "include": [
+        "src",
```

Comment:
dlaczego tak dodałeś?, to co jest linijke nizej nie dziala?

Reply:
nie sprawdzalem tego. Usuwam na razie i nastepnym razem zrobię normalnie :)